detector/model_detector.go

```diff
@@ -50,4 +50,8 @@ type Detector struct {
 	Teams []string `json:"teams"`
 	// Options that control the appearance of a detector in the SignalFx web UI.
 	VisualizationOptions *Visualization `json:"visualizationOptions,omitempty"`
+	// ID of the parent detector from which this detector is customized and created. This property is required for detectors with detectorOrigin of type AutoDetectCustomization.
+	ParentDetectorId string `json:"parentDetectorId,omitempty"`
+	// Indicates how a detector was created. The possible values are: Standard, AutoDetect, AutoDetectCustomization.You can only use Standard or AutoDetectCustomization to create custom detectors.
```

Comment:
is this going to be documented or tested in this repository?

Reply:
I've updated detector's json files used for testing. As for more testing and documentation that will take place in terraform-provider repo